config.example.wikimedia.yaml

```diff
@@ -39,7 +39,7 @@ wikimedia.org: &wikimedia.org
     - path: projects/wikimedia.org.yaml
       options:
         <<: *default_options
-        pageviews:
+        metrics:
```

Comment:
No, that's not true @milimetric . All this does is it renames an internal options object's property name. I'm not saying we shouldn't do this, I'm saying going through a config change procedure just to change a name that is seen nowhere and affects nothing is futile and a waste of time. Once we have some other config changes to do to RB, I'd be more than happy to include this small change too.